src/main/java/net/imagej/legacy/convert/ResultsTableColumnWrapper.java

```diff
@@ -0,0 +1,345 @@
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+
+package net.imagej.legacy.convert;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+import net.imagej.table.GenericColumn;
+
+import ij.measure.ResultsTable;
+
+/**
+ * Wraps a {@link ij.measure.ResultsTable} column as a {@link GenericColumn}.
+ *
+ * @author Alison Walter
+ */
+public class ResultsTableColumnWrapper extends GenericColumn {
+
+	private final ij.measure.ResultsTable table;
+	private final int col;
+
+	public ResultsTableColumnWrapper(final ij.measure.ResultsTable table,
+		final int col)
+	{
+		this.table = table;
+		this.col = col;
+	}
+
+	@Override
+	public String getHeader() {
+		return table.getColumnHeading(col);
+	}
+
+	@Override
+	@SuppressWarnings("deprecation")
+	public void setHeader(final String header) {
+		table.setHeading(col, header);
+	}
+
+	@Override
+	public boolean isEmpty() {
+		return table.getColumn(col) == null;
+	}
+
+	@Override
+	public boolean contains(final Object o) {
+		if (o instanceof Number) {
+			final double value = ((Number) o).doubleValue();
+			for (int i = 0; i < table.size(); i++) {
+				if (table.getValueAsDouble(col, i) == value) return true;
+			}
+			return false;
+		}
+		else if (o instanceof String) {
+			final String value = ((String) o);
+			for (int i = 0; i < table.size(); i++) {
+				if (table.getStringValue(col, i).equals(value)) return true;
+			}
+			return false;
+		}
+		return false;
+	}
+
+	@Override
+	public Iterator<Object> iterator() {
+		throw new UnsupportedOperationException("iterator()");
+	}
+
+	@Override
+	public Object[] toArray() {
+		final Object[] values = new Object[table.size()];
+		for (int i = 0; i < values.length; i++) {
+			if (checkString(i)) values[i] = table.getStringValue(col, i);
+			else values[i] = table.getValueAsDouble(col, i);
+		}
+		return values;
+	}
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public <T> T[] toArray(final T[] a) {
+		final T[] copy = a.length < table.size() ? (T[]) java.lang.reflect.Array
+			.newInstance(a.getClass().getComponentType(), table.size()) : a;
+		for (int i = 0; i < copy.length; i++) {
+			if (i >= table.size()) copy[i] = null;
```

Comment:
Do you mean bound the `for` loop on `table.size()` directly? What if `a` is both longer than `table.size()` and contains a bunch of random values? In such a case, I thought it would be reasonable to set those values to `null`. But, looking at the javadoc for [`List.toArray(T[] a)`](https://docs.oracle.com/javase/8/docs/api/java/util/List.html#toArray-T:A-) that's not right. If `a` is longer than needed, the value immediately after the end of the `List` is set to `null`. So that's probably what should be done ... I'll fix that!